composer.json

```diff
@@ -5,6 +5,6 @@
   "license": "GPL-2.0-or-later",
   "minimum-stability": "dev",
   "require": {
-    "drupal/scheduler": "^1.1"
+    "drupal/scheduler": "1.x-dev"
```

Comment:
Cannot do that, distribution requires 1.1: https://github.com/thunder/thunder-distribution/blob/8.x-4.x/composer.json#L129

Reply:
Does it conflict? I don't think so

Reply:
Ok, it seems that we get the dev version with this in the distribution. Strange...